### ISSUE TITLE
Encoding issue when sending umlauts to F# interactive

### ISSUE THREAD

**Issue by toburger** [ionide/ionide-vscode-fsharp#1463]
### Describe the bug**

When sending a word with an umlaut to F# interactive the wrong encoding is used, I guess.

### Steps to reproduce

1. Create an F# script file 
2. Create a string with an umlaut: "Wörter"
2. Alt + Enter
3.  F# interactive reads it in as: "W�rter"

### Expected behaviour

Ionide sends the correct umlauts to F# interactive.

### Screenshots

![image](https://user-images.githubusercontent.com/728782/103167547-2a472a80-482c-11eb-94a0-f3106a639297.png)

## Machine info
 - OS: Windows
 - .NET SDK version: 5.0.100
 - Ionide version: 5.0.3

**Comment by toburger** [comment]
I've also noticed that when I write the string directly into F# interactive the same issue occurs.
So it seems that it has something to do with how the F# interactive process gets launched inside Visual Studio Code?

(I've launched Python interactive in the same session which doesn't have those issues)
![image](https://user-images.githubusercontent.com/728782/103167693-a9892e00-482d-11eb-97f6-4485b985ba42.png)

Is this a `dotnet fsi` bug?
